lib/check_statistics.rb

```diff
@@ -55,7 +55,12 @@ def number_of_newsletters_sent(team_id, start_date, end_date, language)
           nil
         end
       end.reject{ |t| t.blank? }.collect{ |t| Time.parse(t.to_s) }.select{ |t| t >= start_date && t <= end_date }.collect{ |t| t.to_s }.uniq
-      times.size
+      old_count = times.size
+
+      newsletter = TiplineNewsletter.where(team_id: team_id, language: language).last
```

Comment:
**praise** soooo much simpler

**question** we should only have one month where we expect to tally the old + new (this one), right? other than that, we'd expect either old_count (if we're regenerating past months) or new_count (for anything after, say, june)?

Reply:
Correct! We can clean it up later.